plugins/rest/plugin_impl_rest.go

```diff
@@ -70,6 +76,10 @@ type Deps struct {
 	VPP          vpp.API
 }
 
+type indexes struct {
```

Comment:
If we are going to have data structure for the index page it should be called `IndexPage`, since the *indexes* doesn't make much sense here and actually might just be confused with VPP indexes.